### DISCUSSION TITLE
Need Pagespeed Module for breeze theme

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#224] [1. Help]
Hello @vovayatsyuk 

I am trying to configure swissuplabs/pagespeed extension but I am not able to install neither using composer nor directly zip file in my breeze module or theme and getting error for spattie/imageoptimizer, minify_html ,interceptor class not found and keys are not valid so can you help me out for getting module installation for same. Swissup/codemirror and  swissup/image module error is also getting.

Thanks in advance

**Comment by roma84** [comment]
Hello @JamesDenish, 
if you have issues installing Pagespeed, please contact our support.
